website/docs/cluster-management/profiles.mdx

```diff
@@ -78,6 +78,54 @@ spec:
 ...
 ```
 
+#### (Optional) Using a helm chart from a remote public/private repository
+The helm releases with the profiles can be added in a remote repository which can be referenced using a HelmRepository resource.The repository can be either public or private, although extra steps are required to use the private repo.
+
+In this example a public repo and branch is referenced directly where the helm releases are
+```yaml title="HelmRepository.yaml"
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: weaveworks-charts
+  namespace: flux-system
+spec:
+  interval: 1m
+  url: https://raw.githubusercontent.com/weaveworks/weave-gitops-profile-examples/gh-pages
+```
+
+To be able to use private repositories with restricted access, a secret can be used and synced to the target leaf cluster using `SecretSync`. 
+
+The secret is referenced in the SecretSync as `spec.secretRef` and the labels of the target leaf cluster are added for the syncer to match clusters against those labels using  `spec.clusterSelector.matchLabels`.
+```yaml title="SecretSync.yaml"
+apiVersion: capi.weave.works/v1alpha1
+kind: SecretSync
+metadata:
+  name: my-dev-secret-syncer
+  namespace: flux-system
+spec:
+  clusterSelector:
+    matchLabels:
+      weave.works/capi: bootstrap
+  secretRef:
+    name: weave-gitops-enterprise-credentials
+  targetNamespace: my-namespace
```

Comment:
i could think that 

https://github.com/weaveworks/weave-gitops/pull/3620/files#diff-230071a37d2ecb2a12d9cbc206c44ff1750df80f03b9826dd7854c711ac82991R111

might need to be the same as https://github.com/weaveworks/weave-gitops/pull/3620/files#diff-230071a37d2ecb2a12d9cbc206c44ff1750df80f03b9826dd7854c711ac82991R120

so source controller can access the secret?